hypershift-operator/controllers/sharedingress/sharedingress_controller.go

```diff
@@ -198,7 +199,18 @@ func (r *SharedIngressReconciler) reconcileRouter(ctx context.Context, pullSecre
 		}
 	}
 
-	// TODO(alberto): set PDBs.
+	routerDeploymentOwnerRef := config2.OwnerRefFrom(deployment)
+
+	pdb := RouterPodDisruptionBudget()
+	if result, err := r.createOrUpdate(ctx, r.Client, pdb, func() error {
+		ReconcileRouterPodDisruptionBudget(pdb, routerDeploymentOwnerRef)
```

Comment:
Should we also increase the deployment replicas to 2?

Reply:
done